apple/RNGHStylusData.h

```diff
@@ -0,0 +1,82 @@
+//
+//  RNGHStylusData.h
+//  Pods
+//
+//  Created by Michał Bert on 18/09/2024.
+//
+
+#ifndef RNGHStylusData_h
+#define RNGHStylusData_h
+
+@interface StylusData : NSObject
```

Comment:
```suggestion
@interface RNGHStylusData : NSObject
```

Everything in ObjC is in the global scope

Reply:
Done in [b359bc4](https://github.com/software-mansion/react-native-gesture-handler/pull/3113/commits/b359bc4b89d43d0cfb1bce90c1086792e8d5485e)